backend/app.py

```diff
@@ -1,26 +1,28 @@
-from tkinter import N
-from urllib import response
-from flask import Flask
+from flask import Flask, jsonify, request
+import json 
+
 import db
 
 app = Flask(__name__)
 
+
 @app.route("/")
 def hello_world():
     return "<p>Hello, World!</p>"
 
-@app.route('/surveys')
+
+@app.route("/api/get/surveys", methods=['GET'])
```

Comment:
nit: not the best practice to put the method name in the route. can leave it but its a minor change